crates/fuel-core/src/executor.rs

```diff
@@ -967,22 +967,22 @@ where
         // Check or set the executed transaction.
         match execution_kind {
             ExecutionKind::Validation => {
-                // ensure tx matches vm output exactly
+                // ensure tx matches VM output exactly
                 if &tx != checked_tx.transaction() {
                     return Err(ExecutorError::InvalidTransactionOutcome {
                         transaction_id: tx_id,
                     })
                 }
             }
             ExecutionKind::DryRun | ExecutionKind::Production => {
-                // malleate the block with the resultant tx from the vm
+                // malleable the block with the resultant tx from the VM
```

Comment:
"malleable" isn't a verb. I'm not sure what "malleate" means either. Maybe we choose a new word.

Reply:
I would suggest "mutate" here